### DISCUSSION TITLE
Do not see html in frame_card

### DISCUSSION THREAD

**Discussion by meganjkurka** [h2oai/wave#710] [Q&A]
#### Wave SDK Version, OS

0.12.1, Mac

#### Actual behavior

HTML content not shown, just title
<img width="1327" alt="Screen Shot 2021-03-29 at 3 23 15 PM" src="https://user-images.githubusercontent.com/20548431/112888769-c5a5a100-90a2-11eb-9700-0c7cfd1dfb29.png">

#### Expected behavior

Expect to see html content.

#### Steps To Reproduce

```
from h2o_wave import ui, data, Q, main, app
@app('/')
async def serve(q: Q):
    q.page['meta'] = ui.meta_card(box='', layouts=[
        ui.layout(
            breakpoint='xl',
            width='1300px',
            zones=[ui.zone('content'),]
        )
    ])
    html = '''
<!DOCTYPE html>
<html>
<body>
  <h1>Hello World!</h1>
</body>
</html>
'''
    q.page['content'] = ui.frame_card(box='content',
                                      title='TITLE',
                                      content=html,
                                      )
    await q.page.save()
```

**Comment by lo5** [comment]
The `content` zone needs a height defined. Browsers don't resize HTML frames automatically based on their content.

**Comment by lo5** [comment]
Related: Use a `markup_card()` instead of a `frame_card()` if you want the cards to auto-resize based on content: https://h2oai.github.io/wave/docs/cards#content
